### ISSUE TITLE
TS:1.0 – Stitches.PropertyValue<'flexGrow'> error with string

### ISSUE THREAD

**Issue by peduarte** [modulz/stitches#736]
We have an `fg` utils like this:
```
fg: (value: Stitches.PropertyValue<'flexGrow'>) => ({ flexGrow: value })
```

Using it like this errors:
```
{
  fg: '1'
}
```

**Comment by jonathantneal** [comment]
Fixed in 08c563516301a4e0f98c45290787715385586a15

**Comment by peduarte** [comment]
Fixed in V1. Please refer to [migration guide](https://stitches.dev/blog/migrating-from-beta-to-v1) and [changelog](https://stitches.dev/blog/stitches-1.0.0)
